text/0202-events-and-logs-api.md

```diff
@@ -0,0 +1,193 @@
+# Introducing Events and Logs API
+
+In this OTEP we introduce an Events and Logs API that is based on the OpenTelemetry Log signal. The Events here refer to the independent Events and not to be confused with Span Events which occur only in the context of a span. In OpenTelemetry's perspective Log Records and Events are different names for the same concept - however, there is a subtle difference in how they are represented using the underlying data model that is described below. We will describe why the existing Logging APIs are not sufficient for the purpose of creating events.  It will then be evident that we will need an API in OpenTelementry for creating events.
+
+The Logs part of the API introduced here is supposed to be used only by the Log Appenders and end-users should continue to use the logging APIs available in the languages.
+
+## Subtle differences between Logs and Events
+
+Logs have a mandatory severity level as a first-class parameter that events do not have, and events have a mandatory name that logs do not have. Further, logs typically have messages in string form and events have data in the form of key-value pairs. It is due to this that their API interface requirements are slightly different.
+
+## Who requires Events API
+
+Here are a few situations that require recording of Events, there will be more.
+
+- RUM events (Client-side instrumentation)
```

Comment:
I assume this isn't covering the case of Google Analytics tracking events?

Reply:
It is for recording any standalone events that may occur even when there is no span in progress.